src/doc_table.c

```diff
@@ -146,16 +146,19 @@ int DocTable_SetSortingVector(DocTable *t, t_docId docId, RSSortingVector *v) {
   if (!dmd) {
     return 0;
   }
-
+  
+  //LCOV_EXCL_START
   /* Null vector means remove the current vector if it exists */
-  if (!v) {
+  /*if (!v) { 
```

Comment:
I think its safer not to remove this code

Reply:
It is called from 1 place and checked right before so at the moment, there is no chance.
Do you expect it to be used elsewhere?

Reply:
I think the api was meant to allow it so even if its not used I think its better to leave it